Itvo topic (#516)
ubvu/open-handbook: topics/itvo.qmd

---
title: IT for Research (ITvO)
categories: [Trainings and Knowledge Items]
---

ITvO (IT voor Onderzoek, IT for Research) is a dedicated team within the university IT department, focused on supporting researchers with IT solutions. We bridge the gap between research needs and IT possibilities, making sure researchers have access to the right tools, infrastructure, and expertise.

## Our Services

> - [SciStor](./scistor.qmd): Scientific storage solution of Vu Amsterdam. Data is stored on campus in our Data Centre.

Reviewer: ```suggestion
- [SciStor](./scistor.qmd): Scientific storage solution of VU Amsterdam. Data is stored on campus in our Data Centre.
```

> - [SciCloud](./scicloud.qmd): A flexible platform where researchers can create and manage virtual machines for diverse projects.

Reviewer: ```suggestion
- [SciCloud](../topics/scicloud.qmd): A flexible platform where researchers can create and manage virtual machines for diverse projects.
```

- Housing and hosting: If a virtual machine is not suitable, we can host a physical server for you in our Data Centre.

> - [Ada HPC](./ada.qmd): High Performance Computing cluster of the VU, for compute-intensive research tasks.

Reviewer: ```suggestion
- [Ada HPC](../topics/ada.qmd): High Performance Computing cluster of VU Amsterdam, for compute-intensive research tasks.
```

> - [Consultancy](./consultancy.qmd): Personalized guidance to help researchers navigate IT options, optimize workflows, and solve technical challenges.

Reviewer: ```suggestion
- [Consultancy](../topics/itvo-consultancy.qmd): Personalized guidance to help researchers navigate IT options, optimize workflows, and solve technical challenges.
```
